ccd-definition/CaseEventToFields/CareSupervision/manageOrders/7_otherDetailsSection.json

```diff
@@ -0,0 +1,90 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "manageOrders",
+    "CaseFieldID": "otherDetailsSectionHeader",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "READONLY",
+    "PageID": "OtherDetails",
+    "PageDisplayOrder": 8,
+    "PageColumnNumber": 1,
+    "PageShowCondition": "orderTempQuestions.whichOthers=\"YES\" AND pageShow=\"YES\"",
```

Comment:
do you need pageShow? can use the logic to change orderTempQuestions.whichOthers
See CloseCaseBlockPrePopulator             return Map.of("orderTempQuestions", orderTempQuestions.toBuilder().closeCase("NO").build());